docs/reference-manual/native-image/Bundles.md

```diff
@@ -186,24 +207,83 @@ You successfully rebuilt the application from the bundle with debug info enabled
 
 The full option help of `--bundle-apply` shows a more advanced use case that will be discussed [later](#combining---bundle-create-and---bundle-apply) in detail:
 ```
---bundle-apply=some-bundle.nib
+--bundle-apply=some-bundle.nib[,dry-run][,container[=<container-tool>][,dockerfile=<Dockerfile>]]
                       an image will be built from the given bundle file with the exact same
                       arguments and files that have been passed to native-image originally
                       to create the bundle. Note that if an extra --bundle-create gets passed
                       after --bundle-apply, a new bundle will be written based on the given
-                      bundle args plus any additional arguments that haven been passed
+                      bundle args plus any additional arguments that have been passed
                       afterwards. For example:
                       > native-image --bundle-apply=app.nib --bundle-create=app_dbg.nib -g
                       creates a new bundle app_dbg.nib based on the given app.nib bundle.
                       Both bundles are the same except the new one also uses the -g option.
 ```
 
+
+### Building in a Container
+
+Another addition to the `--bundle-create` and `--bundle-apply` options, as mentioned above, is to perform image building inside a container image.
+This ensures that during the image build `native-image` can not access any resources that were not explicitly specified via the classpath or module path.
+Modify the `--bundle-create` argument in the `native-maven-plugin` configuration to `<buildArg>--bundle-create,container<buildArg>`.
+This still creates the same bundle as before. However, a container image is built and then used for building the native image executable.
+If the container image is newly created, you can also see the build output from the container tool. The name of the container image is the hash of the used Dockerfile.
+If the container image already exists you will see the following line in the build output instead:
+
+```shell
+Native Image Bundles: Reusing container image c253ca50f50b380da0e23b168349271976d57e4e.
+```
+
+For building in a container you require either _podman_ or _rootless docker_ to be available on your system.
+Additionally, building in a container is currently only supported for Linux, if using any other OS native image will not create and use a container image.
```

Comment:
Use `... only supported for Linux. Using any other OS native image ...`

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/83db0c17b7812eac5e15f70d23d47ae0fe14cd44